lib/govuk_message_queue_consumer/test_helpers/test_consumer.rb

```diff
@@ -3,5 +3,13 @@ class TestConsumer < Consumer
     def publish_message(payload, options)
       exchange.publish(payload, options)
     end
+
+    # call after integration tests finish
+    def purge_and_unbind_queue(routing_key)
+      # purge all messages from the queue and
+      # unbind the queue with the given routing_key
+      queue.purge
+        .unbind(exchange, {routing_key: routing_key})
```

Comment:
The test environment uses a different rabbitmq-user from development I believe, so maybe the error could be casued by dev user trying to delete a queue owned by test-user?